### ISSUE TITLE
No previews work

### ISSUE THREAD

**Issue by kghose** [microsoft/vscode#98314]
Issue Type: <b>Bug</b>

Disable all extentensions. Try markdown preview. 
```
command 'markdown.showPreviewToSide' not found
```

This happened after updating to 1.45.1. All of my extensions previews stopped working. I disabled all extensions and tried the markdown preview, and got this.

Possibly related to #98306  , #72873

VS Code version: Code 1.45.1 (5763d909d5f12fe19f215cbfdd29a91c0fa9208a, 2020-05-14T08:33:47.663Z)
OS version: Darwin x64 19.4.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-5257U CPU @ 2.70GHz (4 x 2700)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|5, 6, 5|
|Memory (System)|16.00GB (0.09GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>Extensions: none

**Comment by mjbvz** [comment]
Have you disabled the builtin markdown extension? In the extensions view, search `@builtin markdown` to check this

**Comment by kghose** [comment]
Hi @mjbvz thanks for responding. The problem has resolved itself. For a while despite me uninstalling all extensions, reinstalling etc. no preview, builtin or otherwise, would work. Then, just as suddenly as the problem started, it disappeared. I did not reboot the mac or anything. I did delete all VS code state at some point, but the problem persisted after that. I can not think of why this would have happened.
